Source/AliveLib/MainMenu.cpp

```diff
@@ -1672,21 +1672,271 @@ void MainMenuController::tGame_BackStory_Or_NewGame_Unload_4D1BE0()
     pResourceManager_5C1BB0->LoadingLoop_465590(FALSE);
 }
 
+char MainMenuController::sub_convert_iso_FileName_4D1660(char *a1, char *a2)
+{
+    char result = *a2;
```

Comment:
this function is quite messy its probably easier to rewrite it. I think its just copying the input to the output up to the first found ; ?